src/interaction/VSphereSelf.hpp

```diff
@@ -121,7 +121,7 @@ namespace espressopp {
         real energy;
         energy  =   e1*pow(1.0L*M_4PI3*sigma2, 1.0L*mth)       \
                   + a1*NbNbNb/(sigma2*sigma2*sigma2) \
-                  + a22dNb*sigma2;
+                  + a2dNb*sigma2;
```

Comment:
If I look on formula (http://espressopp.github.io/espressopp.interaction.VSphereSelf.html) this should be 2*a_2/Nb so this part was correct (or if not then please update VSphereSelf.py to make it consistent and preferable add reference to some source where this potential was firstly introduce)
Another issue, is this sigma squared ? again in the doc, there is only sigma_2 and not sigma^2